node_modules/oae-core/signin/css/signin.css

```diff
@@ -74,3 +74,97 @@ body.oae-page-signup #signin-modal-signup, body.oae-page-signup #signin-modal-re
 #signin-modal-resetpassword > button {
     float: right;
 }
+
+select#form_field, .select2-container--default {
+	width: 100% !important;
```

Comment:
Are all the `!important`s necessary?

Reply:
Apparently, yes. There's a comment somewhere stating explicitly that `!important` should be kept, but I'm not an expert in CSS to assess if that's really the case. Is it a bad practice or something?

Reply:
I think it's generally considered pretty poor practice since it kind of overrides the 'cascading' part of CSS. Can't be helped though I guess?